runtime/calamari/src/assets_config.rs

```diff
@@ -175,3 +182,17 @@ impl pallet_manta_pay::Config for Runtime {
     type AssetConfig = CalamariAssetConfig;
     type PalletId = MantaPayPalletId;
 }
+
+parameter_types! {
+    pub const MantaSbtPalletId: PalletId = MANTA_SBT_PALLET_ID;
+}
+
+impl pallet_manta_sbt::Config for Runtime {
+    type Event = Event;
+    type PalletId = MantaSbtPalletId;
+    type Currency = Balances;
+    type MintsPerReserve = ConstU16<5>;
+    type ReservePrice = ConstU128<{ 100 * KMA }>;
+    type SbtMetadataBound = ConstU32<300>;
+    type WeightInfo = ();
```

Comment:
change to manta-sbt weights

Reply:
Added, still need to actually benchmark it using CI, which will be done on release.